server/Entities/RepositoryContext.cs

```diff
@@ -0,0 +1,31 @@
+using Entities.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Entities
+{
+    public class RepositoryContext : DbContext
+    {
+
+        public DbSet<User> Users { get; set; }
+
+        public RepositoryContext(DbContextOptions options) : base(options)
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder options)
+            => options.UseSqlServer(
+                "Server=192.168.1.128,51528\\SQLEXPRESS;Database=master;User Id=sa;Password=server;"
+                );
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>().HasData(
+                new User { Id = -1, Email = "test1@users.com", Password = CryptoUtils.HashPassword("test1") },
```

Comment:
you have this in migration, don't you?

Reply:
Not sure do I understand the question correctly, but generated Migration class contains Instert statement for all seeded data:
```
migrationBuilder.InsertData(
                table: "users",
                columns: new[] { "Id", "Email", "Password" },
                values: new object[,]
                {
                    { -1, "test1@users.com", "d53tbpepSAOL9x9Xo2ea/LLhaJSgmyO0CCXFrYcdk66s3icf5o3lPGejbtagCjdD" }
```